test/snapshots/unparser/corpus/semantic/dstr.txt

```diff
@@ -198,11 +198,11 @@ ProgramNode(0...608)(
        )
      ),
      StringConcatNode(552...560)(
-       StringNode(552...554)((552...553), (553...553), (553...554), ""),
-       StringConcatNode(555...560)(
-         StringNode(555...557)((555...556), (556...556), (556...557), ""),
-         StringNode(558...560)((558...559), (559...559), (559...560), "")
-       )
+       StringConcatNode(552...557)(
```

Comment:
Not sure I follow why this changed? It was incorrect before?

Reply:
It was not necessarily incorrect so much as just another way to represent it (right-associative vs left-associative). It's moot at the end of the day because everyone that consumes these nodes is going to concat them together anyway.

It's the difference between parsing:

```ruby
"a" + "b" + "c"
```

as:

```ruby
("a" + "b") + "c"
```

vs

```ruby
"a" + ("b" + "c")
```

Because of the way the loop works, it happened to change the associativity, but I don't think it matters for our use case. It might even be better, because it's more consistent with how calls work.